commands/databases_test.go

```diff
@@ -36,6 +36,11 @@ var (
 		Hour: "10:00",
 	}
 
+	testBackUpRestore = &godo.DatabaseBackupRestore{
+		DatabaseName:    "sunny-db-cluster",
+		BackupCreatedAt: time.Now().UTC().Format("2006-01-02T15:04:05Z07:00"),
```

Comment:
You can use a predefined format here.

```suggestion
		BackupCreatedAt: time.Now().UTC().Format(time.RFC3339),
```